REFERENCE.md

```diff
@@ -2316,6 +2316,7 @@ Struct[{
     Optional['AmbientCapabilities']       => Variant[Pattern[/^CAP_[A-Z_]+$/],Array[Pattern[/^CAP_[A-Z_]+$/],1]],
     Optional['User']                      => String[1],
     Optional['Group']                     => String[1],
+    Optional['WorkingDirectory']          => String,
```

Comment:
Without an explicit minimum length, I trend to think that it was forgotten.  So I suggest:

```suggestion
    Optional['WorkingDirectory']          => String[0],
```

Reply:
You need to allow `WorkingDirectory=''` to allow people to set back to the default.
Folk could explicitly set back to the default I guess but other similar directives support `''`.

Reply:
Sorry misinterpreted - Explicit 0 ?  that's the default anyway surely?